dkg/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "dkg"
```

Comment:
Any chance this would get published? Any better names perhaps?

Reply:
We could rename them to something like frost-<project>-demo or something like that. But this won't get published so I don't think it's necessary. I'm going to merge this PR and we can discuss independently.